campaign/campaign_main/campaign_10_2.py

```diff
@@ -24,44 +24,90 @@
     10 10 10 10 10 10 10 10
 """
 MAP.spawn_data = [
-    {'battle': 0, 'enemy': 4},
-    {'battle': 1, 'enemy': 2},
-    {'battle': 2, 'enemy': 1},
-    {'battle': 3, 'enemy': 1},
-    {'battle': 4, 'enemy': 1},
-    {'battle': 5},
-    {'battle': 6, 'boss': 1},
+    {"battle": 0, "enemy": 4},
+    {"battle": 1, "enemy": 2},
+    {"battle": 2, "enemy": 1},
+    {"battle": 3, "enemy": 1},
+    {"battle": 4, "enemy": 1},
+    {"battle": 5},
+    {"battle": 6, "boss": 1},
 ]
-A1, B1, C1, D1, E1, F1, G1, H1, \
-A2, B2, C2, D2, E2, F2, G2, H2, \
-A3, B3, C3, D3, E3, F3, G3, H3, \
-A4, B4, C4, D4, E4, F4, G4, H4, \
-A5, B5, C5, D5, E5, F5, G5, H5, \
-A6, B6, C6, D6, E6, F6, G6, H6, \
-    = MAP.flatten()
+(
+    A1,
+    B1,
+    C1,
+    D1,
+    E1,
+    F1,
+    G1,
+    H1,
+    A2,
+    B2,
+    C2,
+    D2,
+    E2,
+    F2,
+    G2,
+    H2,
+    A3,
+    B3,
+    C3,
+    D3,
+    E3,
+    F3,
+    G3,
+    H3,
+    A4,
+    B4,
+    C4,
+    D4,
+    E4,
+    F4,
+    G4,
+    H4,
+    A5,
+    B5,
+    C5,
+    D5,
+    E5,
+    F5,
+    G5,
+    H5,
+    A6,
+    B6,
+    C6,
+    D6,
+    E6,
+    F6,
+    G6,
+    H6,
+) = MAP.flatten()
```

Comment:
I don't think you have read the development docs or any lines of code in ALAS. If you did, you'll know none of these changes will improve readability.

Reply:
The process of reformatting is fully automatic, sorry if I didn't check every single file.
Overall, except this problem with the readability of the campaign levels, to me the code looks better (better indents, string management, variables and so on).
Other than being my personal opinion, remember that Black is used at an enterprise level.

I would consider leaving unchanged the campaign .py files, and merge the rest.
I understand that this is a 955 files changed commit, but the reason I did this is because I was surprised you didn't use Black before.

This said, don't judge people based on PRs history. I wouldn't be here if I didn't want to give a hand.